common/app/views/fragments/amp/stylesheets/main.scala.html

```diff
@@ -5,279 +5,16 @@
-        color: #767676;
-    }
-
-    .ad-slot--paid-for-badge__header {
-        margin: 3px 0 0;
-    }
-
-    .quoted__contents {
-        font-style: italic;
-    }
-
-    .u-cf:after, .main-body:after {
-        clear: both;
-        content: "";
-        display: block;
-    }
-
-    .u-cf:after, .u-cf:before {
-        content: '';
-        display: table;
-    }
-
-    .caption {
-        padding-top: 0.5rem;
-        padding-bottom: 16px;
-    }
-
-    .content__dateline {
-        padding: 0.125rem 0 0.5rem;
-        margin: 0;
-    }
-
-    .caption, .content__dateline, .ad-slot--paid-for-badge__header {
-        font-size: 0.75rem;
-        line-height: 1rem;
-    }
-
-    .content__meta-container {
-        border-top: 0.0625rem dotted #dfdfdf;
-        border-bottom: 0.0625rem dotted #dfdfdf;
-        margin-bottom: 0.75rem;
-    }
-
-    .submeta {
-        border-top: 0.0625rem dotted #dfdfdf;
-        padding: 6px 0 12px;
-        clear: both;
-    }
-
-    .u-underline {
-        text-decoration: none;
-        border-bottom: 0.0625rem solid #dcdcdc;
-        color: #005689;
-    }
-
-    @* Dropcap styling *@
-    .drop-cap__inner {
-        float: left;
-        display: inline-block;
-        vertical-align: text-top;
-        font-size: 3rem;
-        line-height: 2.5rem;
-    }
-
-    .drop-cap {
-        float: left;
-        display: inline-block;
-        text-transform: uppercase;
-        -webkit-box-sizing: border-box;
-        -moz-box-sizing: border-box;
-        box-sizing: border-box;
-        height: 2.5rem;
-        padding-top: 0.0625rem;
-        margin-right: 0.25rem;
-    }
-
-    @* Bullet Points related styling below *@
-    .bullet {
-        font-size: 0.00625rem;
-        color: transparent;
-    }
-
-    .bullet:before, .tonal__standfirst ul>li:before, .from-content-api ul>li:before {
-        display: inline-block;
-        content: '';
-        -webkit-border-radius: 0.375rem;
-        border-radius: 0.375rem;
-        height: 0.75rem;
-        width: 0.75rem;
-        margin-right: 0.125rem;
-        background-color: #bdbdbd;
-    }
-
-    .from-content-api>ol>li, .from-content-api>ul>li {
-        position: relative;
-        margin-bottom: .8em;
-    }
-
-    .content__standfirst>ul, .content__standfirst>ol, .from-content-api>ul {
-        margin: .8em 0 0;
-        padding: 0;
-        list-style: none;
-    }
-
-    .fc-container__header__description {
-        font-size: 1rem;
-        line-height: 1.25rem;
-        margin-top: -1rem;
-        padding-bottom: 1rem;
-        color: #767676;
-    }
-
-    @* temporarily hiding these until after launch *@
-    .inline-expand-image, .content__dateline-lm {
-        display: none;
-    }
-
-    @fragments.amp.stylesheets.fonts()
+    @* TODO: these two stylesheets should be refactored into SCSS *@
```

Comment:
@lps88 without having looked too far into it, my instinct would be to add a classname based on brand colour and have css for them all. That is how we do it with all our various tones.